package.json

```diff
@@ -32,7 +32,7 @@
     "build:dockerimage": "docker build -t mydockerimage  --build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) . ",
     "build:emscripten": "echo $(pwd) && docker run --rm -v $(pwd):/src    -u $(id -u):$(id -g)  mydockerimage    ./build.sh",
     "copy-files": "copyfiles -u 1 src/xeus_lua.wasm  src/xeus_lua.worker.js src/xeus_lua.js lib",
-    "build:wasm": "jlpm run build:dockerimage && jlpm run build:emscripten && jlpm run copy-files",
+    "build:wasm": "jlpm run build:dockerimage && jlpm run build:emscripten",
```

Comment:
Thinking that we might not need to manually copy the files and `webpack` would take care of bundling them automatically. Let's see what CI says.

Reply:
When the copy-files is removed, I am also puzzled that xeus_lua.js does not end up in the "lib" folder.

Reply:
Gonna have a look locally.